docs/notes-prove-verify.md

```diff
@@ -1,6 +1,392 @@
-In this module, the notes on how the steps 
-the prover must follow to construct
-PLONK proof, are documented.
+In this module, the notes on how the prover 
+algorithm is constructed
+for PLONK, is documented.
 
-Proving steps
-============
\ No newline at end of file
+PLONK proof construction 
+========================
+
+Following on from the generic
+SNARK construction, 
+here we will give the set up
+of a PLONK proof and show 
+which steps need to be satisfied
+to utilise the protocol.
+
+First we will explain the 
+derivation and simplification 
+of the arithmetic circuits. 
+
+PLONK uses both gate constraints 
+and copy constraints, to collect 
+like expressions. Using the same
+example of:
+
+\\[
+\mathbf{W}\_*L* \cdot \mathbf{W}\_*R* = \mathbf{W}\_*O*,
+\\]
+
+We can express multiples of the 
+same wires in or out of the same 
+gate, with the above equation.
```

Comment:
Multiples of  `the same wires`? WTYM?

Reply:
copy constraints is what it means. I can change it to be more clear

Reply:
ohh. Fine. Whatever you think it's more clear.